app/javascript/packs/selectors/questionsSelector.js

```diff
@@ -4,7 +4,18 @@ export function getQuestionsByBuilding(buildingId, state) {
     return state.questions[questionId];
   });
 }
-
+//gets questions based off of the specific building we are viewing
+//entity is simply the part of a portfolio that we are in (portfolios/portfolio_id or buildings/buildingId)
+export function getQuestionsByBuildingNavBar(entity, buildingId, state) {
+    if (entity == 'buildings') {
```

Comment:
Are these checks that we need new selectors for? Is it possible to make these checks inline with our `mapStateToProps` function in `NavigationBarContainer`?